public/styles/normalize.css

```diff
@@ -0,0 +1,415 @@
+html {
```

Comment:
when rabea and I paired she mentioned it is good practice to add this so that you start with a kind of blank state across all browsers...